src/auth/impersonated.ts

```diff
@@ -224,7 +241,9 @@ export class Impersonated extends OAuth2Client implements IdTokenProvider {
       delegates: this.delegates,
       audience: targetAudience,
       includeEmail: options?.includeEmail ?? true,
+      useEmailAzp: options?.includeEmail ?? true,
```

Comment:
Is this field related to the universe domain change?

Reply:
It's recommended for impersonation, and now required for TPC. Other langs have this implemented:
- https://github.com/googleapis/google-auth-library-python/pull/1270
- https://github.com/googleapis/google-auth-library-php/pull/581/files#diff-9caed8dbd080d92079e2315774fe52b2e394a8f34d9e2883031ae9530bb1bbb5R138